girder/models/item.py

```diff
@@ -120,12 +122,8 @@ def load(self, id, level=AccessType.ADMIN, user=None, objectId=True,
                       checking on this resource, set this to True.
         :type force: bool
         """
-        doc = Model.load(self, id=id, objectId=objectId, fields=fields,
-                         exc=exc)
-
-        if not force and doc is not None:
-            self.model('folder').load(doc['folderId'], level, user, objectId,
-                                      force, fields)
+        doc = super(Item, self).load(id, level, user, objectId, force, fields,
+                                     exc)
```

Comment:
That is fine, I guess this will support people mucking with the __mro__ if it ever comes to that :)